lib/travis/build/script/shared/directory_cache/base.rb

```diff
@@ -301,7 +301,11 @@ def update_static_file(name, location, remote_location, assert = false)
               cmd_opts = {retry: true, assert: false, echo: 'Installing caching utilities'}
               if casher_branch == 'production'
                 static_file_location = "https://#{app_host}/files/#{name}".output_safe
-                sh.cmd curl_cmd(flags, location, static_file_location), cmd_opts
+                app_host_flags = flags
+                if Travis::Build.config&.ssl&.verify == false
+                  app_host_flags += '-k'
```

Comment:
I think you need a space before `-`.

Reply:
indeed, works because flags contain already ' ' at the end, to be updated